### PR TITLE
6pm-2-add delete button in room slot table

### PR DESCRIPTION
**Pull request by arunryu**
I this PR, we implemented delete button in front end of room slot table, so we can delete room slot by clicking delete button

### CODE REVIEW

**Review by RaghubirChimni — DISMISSED** [review]
LGTM

File: javascript/src/main/pages/RoomSlots/RoomSlots.js [hunk 1]
```diff
@@ -36,7 +39,7 @@ const RoomSlots = () => {
         <Button className="mb-3" onClick={()=>history.push("/roomslots/new")} >New Room Slot</Button>
       )}
       {(roomSlotList) && (
-        <RoomSlotTable roomSlots={roomSlotList} />
+        <RoomSlotTable roomSlots={roomSlotList} admin={true} deleteRoomSlot={deleteRoomSlot}/>
```
Comment (rowantran) [inline javascript/src/main/pages/RoomSlots/RoomSlots.js:42]:
```suggestion
        <RoomSlotTable roomSlots={roomSlotList} admin={isAdmin} deleteRoomSlot={deleteRoomSlot}/>
```
We should pass the `isAdmin` variable evaluated above into the table.

**Review by rowantran — CHANGES_REQUESTED** [review]
Just a small change needed for the `RoomSlotTable` attributes in the `RoomSlots` page. Looks good other than that, good work!

**Review by RaghubirChimni — APPROVED** [review]
LGTM

**Review by btk5h — APPROVED** [review]

**Pull request merged**